src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -882,4 +882,7 @@ Do you want to continue?</value>
   <data name="WRN_UnableToLoadAnalyzer" xml:space="preserve">
     <value>Unable to load Analyzer assembly {0}: {1}</value>
   </data>
+  <data name="Make_method_synchronous" xml:space="preserve">
```

Comment:
For now, we have a convention used by the rest of the codebase. Let's be consistent for now and if we choose to change the style, we change it at once.